airflow/providers/common/sql/hooks/sql.py

```diff
@@ -528,6 +527,14 @@ def _generate_insert_sql(self, table, values, target_fields, replace, **kwargs)
 
         return self._replace_statement_format.format(table, target_fields, ",".join(placeholders))
 
+    @contextmanager
+    def _closing_supporting_autocommit(self, autocommit: bool = False):
```

Comment:
This function name is weird. I would use something like `_create_auto_connection()`

Reply:
Renamed it to _create_autocommit_connection